hack/update-all.sh

```diff
@@ -54,7 +54,7 @@ if ! $ALL ; then
 	echo "Running in short-circuit mode; run with -a to force all scripts to run."
 fi
 
-kube::util::ensure_godep_version v74
+kube::util::ensure_godep_version v79
```

Comment:
I think it's a mistake and we should update all 3 if you update 1, but I won't block on it.